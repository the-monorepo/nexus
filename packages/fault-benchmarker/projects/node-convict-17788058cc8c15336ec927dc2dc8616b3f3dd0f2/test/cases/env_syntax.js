'use strict';

exports.conf = {
  foo: {
    bar: {
      default: 'a',
      format: ['a', 'b'],
      env: 'BAR'
    }
  }
};

exports.env = {
  BAR: 'c' // hey!  that's not an allowable value!
};
