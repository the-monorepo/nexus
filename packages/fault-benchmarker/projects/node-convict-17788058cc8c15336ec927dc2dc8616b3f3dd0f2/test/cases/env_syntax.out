foo.bar: must be one of the possible values: ["a","b"]: value was "c"
